elements/pfe-progress-steps/src/pfe-progress-steps-item.scss

```diff
@@ -164,6 +164,7 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
       background-color: pfe-local(Color, $region: circle);
       // leave with no fallback so it inherits correctly
       color: pfe-local(Color--active, $region: title);
+      // MP font-weight should be 500 weight
```

Comment:
There's no text in the __circle--outer element anyway, do we need color or font-weight at all?